packages/roosterjs-editor-dom/lib/style/getBiggestZIndex.ts

```diff
@@ -0,0 +1,22 @@
+import getTagOfNode from '../utils/getTagOfNode';
+
+/**
+ * Search through from element to it's root for the biggest z-index value
+ * @param element the parent element
+ * @returns
+ */
+export default function getBiggestZIndex(element: HTMLElement) {
```

Comment:
This function is now only used by image edit plugin, so let's put it there and no need to export, to reduce the size of dom package